client/analytics/components/leaderboard/index.js

```diff
@@ -27,6 +28,8 @@ import ReportError from '../report-error';
 import sanitizeHTML from '../../../lib/sanitize-html';
 import './style.scss';
 
+const Text = TextComponent || __experimentalText;
```

Comment:
If it's not possible to do something like this with Webpack, what do you think about creating a new file to house these fallbacks so they don't have to be implemented everywhere?

Something like.. `import { Text } from 'client/experimental-components';`?

Reply:
That's a much better idea 🤦 

I'll check into Webpack first to see if there's a workaround for something like this.